### PR TITLE
Use proper Google Analytics id

### PR DESCRIPTION
**Pull request by NicolasHug**
This PR fixes the GA id and relies on `pytorch-sphinx-theme`  to set the GA script instead of hard-coding it (this is supported since https://github.com/pytorch/pytorch_sphinx_theme/pull/110 was merged).

Similar PRs were opened and merged in torchchvision/audio/text, e.g.: https://github.com/pytorch/vision/pull/3700

CC @brianjo

### CODE REVIEW

**Comment by NicolasHug** [comment]
The GA script is properly included in the header of the rendered html docs, see e.g. the html source of https://12665473-65600975-gh.circle-artifacts.com/0/docs/backends.html

**Review by brianjo — APPROVED** [review]

**Review by ranman — APPROVED** [review]

**Comment by NicolasHug** [comment]
The PR for torchvision has been merged yesterday already - perhaps we could check that we started to get some signals for https://pytorch.org/vision/master/index.html ?

I don't have access to the GA console so I cannot do it myself unfortunately

**Comment by ranman** [comment]
Working on getting you access. I can confirm we can see the data coming in but let's hold off on merging for a few days while we chase down merging the historical GA data.

File: docs/source/_templates/layout.html
Comment (ranman) [inline docs/source/_templates/layout.html:30]: you can remove this line as well

**Comment by ranman** [comment]
@NicolasHug in layout.html you can remove lines 29-49 of analytics code.

**Comment by ranman** [comment]
You can merge this PR with my updated commit after this PR is merged: https://github.com/pytorch/pytorch.github.io/pull/695

**Pull request closed without merging**
